stack.yaml

```diff
@@ -96,6 +96,7 @@ extra-deps:
 - lens-4.15.3                     # not yet in lts-8
 - th-abstraction-0.2.2.0          # needed for lens
 - rocksdb-haskell-1.0.0
+- generic-arbitrary-0.1.0
```

Comment:
More dependencies! Maybe we can get rid of `derive` package then? I think we use them in more or less same way.